docs/docs/03-light-clients/04-wasm/03-integration.md

```diff
@@ -116,7 +116,7 @@ func NewSimApp(
     ctx := app.BaseApp.NewUncachedContext(true, cmtproto.Header{})
 
     // Initialize pinned codes in wasmvm as they are not persisted there
-    if err := wasmkeeper.InitializePinnedCodes(ctx); err != nil {
+    if err := wasmkeeper.InitializePinnedCodes(ctx); err != nil { // appCodec is required in v7
```

Comment:
I will update this when creating the versioned docs for v7.3.x. Thanks for noticing!